examples/env.yml

```diff
@@ -0,0 +1,31 @@
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
+  - pip
+  - pip:
```

Comment:
I use `pip` instead of `conda` because Nebari's conda store doesn't really work well with conda. Tried solving these issues, but in end the just went with pip.

<img width="914" alt="image" src="https://github.com/holoviz/holoviz/assets/15331990/00193866-96b5-4d59-9600-ab44be5b1777">


<img width="1215" alt="image" src="https://github.com/holoviz/holoviz/assets/15331990/b467414c-244c-4124-a7d7-92ecb0c1f5a8">

Reply:
Ok noted, it's kind of weird but well it's how it is.